tests/share/janitor.py

```diff
@@ -0,0 +1,40 @@
+import pytest
```

Comment:
This file should be named `test_janitor.py`. I was confused why travis was passing with the whole janitor module missing...